discovery-infra/tests/base_test.py

```diff
@@ -117,6 +110,12 @@ def get_cluster_func(nodes: Nodes, cluster_config: ClusterConfig = ClusterConfig
                     logging.info(f'--- TEARDOWN --- deleting created cluster {cluster.id}\n')
                     cluster.delete()
 
+    @pytest.fixture
+    def configs(self) -> Tuple[ClusterConfig, TerraformConfig]:
```

Comment:
i must say i don't like this configs provided here. We are blocking oursevlves to use only terraform and the idea was that controller can e changed and config shouldn't be controller specific at least not in base_test

Reply:
You can always create them manually 
```suggestion
    cluster_config, tf_config = ClusterConfig(), TerraformConfig()
```